backend/service/chaos/experimentation/experimentstore/storer.go

```diff
@@ -228,10 +228,11 @@ func (s *storer) GetListView(ctx context.Context) ([]*experimentation.ListViewIt
 		var details string
 		run := ExperimentRun{}
 		config := ExperimentConfig{Config: &any.Any{}}
-		err = rows.Scan(&run.Id, &run.StartTime, &run.EndTime, &run.CancellationTime, &run.creationTime, &config.id, &details)
+		err = rows.Scan(&run.Id, &run.StartTime, &run.EndTime, &run.CancellationTime, &run.CreationTime, &config.id, &details)
```

Comment:
now sure how it continues to work with `*time.Time` as opposed to `*sql.nulltime`